stable_baselines3/common/utils.py

```diff
@@ -271,3 +271,20 @@ def safe_mean(arr: Union[np.ndarray, list, deque]) -> np.ndarray:
     :return:
     """
     return np.nan if len(arr) == 0 else np.mean(arr)
+
+
+def polyak_update(params: Iterable[th.nn.Parameter], target_params: Iterable[th.nn.Parameter], tau: float):
+    """
+    Perform a polyak average update on target_params using params
+
+    :param params: (Iterable[th.nn.Parameter]) parameters to use to update the target params
+    :param target_params: (Iterable[th.nn.Parameter]) parameters to update
+    :param tau: (float) the soft update coefficient ("Polyak update", between 0 and 1)
+    """
+    params = list(params)
+    target_params = list(target_params)
+    if params:
```

Comment:
maybe an assert instead? "assert len(params) > 0" + meaningful error message?

Reply:
Not sure if it's necessary, the if is simply to get the device, but besides that, it isn't necessary.

Reply:
Never mind my previous reply, the if was garbage left from a different implementation.

Reply:
The previous implementation used the first parameter to get the device and create a tensor with `tau`, but this version doesn't need it. I forgot about it and accidentally left it in.